routes/tasks.go

```diff
@@ -0,0 +1,111 @@
+package routes
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/labstack/echo"
+	"net/http"
+
+	m "github.com/kirikami/go_exercise_api/database/models"
+	u "github.com/kirikami/go_exercise_api/utils"
+)
+
+func (h ApiV1Handler) SaveTaskHandler(c echo.Context) error {
+	task := m.Task{}
+
+	if err := c.Bind(&task); err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError)
+	}
+
+	if err := h.DB.Save(&task).Error; err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError)
+	}
+
+	return c.JSON(http.StatusCreated, task)
+}
+
+func (h ApiV1Handler) UpdateTaskHandler(c echo.Context) error {
+	idParam := c.Param("id")
+
+	id, err := u.ParseIdInt64FromString(idParam)
+
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest)
+	}
+
+	task := m.Task{}
+
+	if err := h.DB.First(&task, id).Error; err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError)
+	}
+
+	if err := c.Bind(&task); err != nil {
+		c.Logger().Error(err)
+		return echo.NewHTTPError(http.StatusInternalServerError)
```

Comment:
Send internal error here and in all similar cases. This is an json api, you should send error message in some defined format, check out git hub API as an example https://developer.github.com/v3/#client-errors